pkg/query-service/README.md

```diff
@@ -6,17 +6,47 @@ Query service is the interface between frontend and databases. It is written in
 - parse response from databases and handle error if any
 - clickhouse response in the format accepted by Frontend
 
+# Complete the clickhouse setup locally.
+https://github.com/jshiwam/signoz/blob/main/CONTRIBUTING.md#to-run-clickhouse-setup-recommended-for-local-development
```

Comment:
@jshiwam This link shouldn't be pointing to your fork.

Reply:
Will fix it and update the PR.

Reply:
Done!!